### PR TITLE
tldr : add korean translation

### PR DESCRIPTION
**Pull request by Proscream**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.ko/common/tldr.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# tldr
+
+> 간단한 매뉴얼 페이지. 더 많은 정보 : [https://tldr.sh](https://tldr.sh/).
```
Comment (einverne) [inline pages.ko/common/tldr.md]:
```suggestion
> 간단한 매뉴얼 페이지. 
> 더 많은 정보 : <https://tldr.sh>.
```
Keep consistent with the whole project.

Comment (Proscream) [reply pages.ko/common/tldr.md]: Thank u! I'll reflect it

File: pages.ko/common/7z.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# 7z
+
+> 높은 압축률을 보여주는  파일 압축 프로그램. 
+>
```
Comment (einverne) [inline pages.ko/common/7z.md]: There is no need to put an empty line here.

File: pages.ko/common/7z.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+> 더 많은 정보: https://www.7-zip.org/.
+
+- 파일 또는 디렉토리 압축하기:
+
+`
+7z a {{archived.7z}} {{path/to/file_or_directory}}
+`
```
Comment (einverne) [inline pages.ko/common/7z.md]: Put the command into one line like other page.

File: pages.ko/common/7z.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+# 7z
+
+> 높은 압축률을 보여주는  파일 압축 프로그램. 
+>
+> 더 많은 정보: https://www.7-zip.org/.
```
Comment (einverne) [inline pages.ko/common/7z.md]:
```suggestion
> 더 많은 정보: <https://www.7-zip.org/>.
```

File: pages.ko/common/7z.md (hunk 3 above)
Comment (Proscream) [reply pages.ko/common/7z.md]: You mean `7z a {{archived.7z}} {{path/to/file_or_directory}}` in one line?

**Comment by Proscream** [comment]
@einverne  Probably changed everything for your request isnt it?.

**Comment by sbrl** [comment]
Are we ok to merge @einverne?

**Pull request merged**